packages/@snowpack/plugin-postcss-transform/plugin.js

```diff
@@ -0,0 +1,21 @@
+const execa = require('execa');
+
+module.exports = function postcssTransformPlugin(_, options) {
+  return {
+    name: '@snowpack/postcss-transform',
+    async transform({fileExt, contents}) {
+      const {input = ['.css'], config, sourceMaps = true} = options;
+      if (!input.includes(fileExt)) return;
+
+      const flags = [];
+      if (config) flags.push(`--config ${config}`);
+
+      const {stdout} = await execa('postcss', flags, {
```

Comment:
We want to use the PostCSS CLI because a) it becomes a dependency that the user version-manages, not Snowpack (we don’t want to lock users to an old version), and b) it lets PostCSS be more intelligent with how it loads its own configuration and plugins.